### PR TITLE
Add article button

### PR DESCRIPTION
**Pull request by Byambaa0325**
Added a button to add an article to the forum it was clicked from

### CODE REVIEW

File: src/main/webapp/Forum-id.jsp [hunk 1]
```diff
@@ -37,6 +37,10 @@
 <body>
 <h1><%=forum.getTitle()%></h1>
 <hr>
+  <form action="/createTrail.html" method="GET" id="AddArticle">
```
Comment (huygaa11) [inline src/main/webapp/Forum-id.jsp:40]: Why is adding an article "GET" method?

File: src/main/webapp/js/create-trails.js [hunk 2]
```diff
@@ -67,6 +69,9 @@ $(window).load(function(){
     params.append('tags', tags);
     params.append('body', body);
     params.append('coordinates', newCoordinates_poly);
+    if(forum !=""){
+      params.append("forum", forum.slice(forum.indexOf("country=")+8));
```
Comment (huygaa11) [inline src/main/webapp/js/create-trails.js:75]: add a comment on what's happening here.

File: src/main/webapp/Forum-id.jsp (hunk 1 above)
Comment (Byambaa0325) [reply src/main/webapp/Forum-id.jsp:40]: It is for directing to article creating page not creating an article. By passing origin of the click, the created article will be added to only the origin forum.

File: src/main/webapp/js/create-trails.js (hunk 2 above)
Comment (Byambaa0325) [reply src/main/webapp/js/create-trails.js:75]: got it

**Pull request merged**